dcpy/library/templates/fbop_corrections.yml

```diff
@@ -3,30 +3,27 @@ dataset:
   acl: public-read
   source:
     url:
-      path: .library/upload/fbop_corrections.csv
+      path: s3://edm-recipes/inbox/fbop_corrections/{{ version }}/fbop_corrections.csv
     options:
       - "AUTODETECT_TYPE=NO"
       - "EMPTY_STRING_AS_NULL=YES"
-      - "X_POSSIBLE_NAMES=longitude"
-      - "Y_POSSIBLE_NAMES=latitude"
     geometry:
-      SRS: EPSG:4326
-      type: POINT
+      SRS: null
```

Comment:
What happened to points?

Reply:
oops I should have explained in the commit message
- this is a csv we manually maintain and it had a column for WKT points
- the website that we check for updates only has addresses, not coordinates
- we geocode all FacDB source data during the build, so we get coordinates from the address

so it seemed bad to propagate old coordinates and bad to have a geometry column at all since the only info we can get from the source is the address

Reply:
Cool, makes sense!